src/installerncc.cpp

```diff
@@ -223,6 +223,19 @@ IPluginInstaller::EInstallResult InstallerNCC::invokeNCC(IModInterface *modInter
       seString = std::wstring() + L"-se \"" + seVersion + L"\"";
     }
   }
+  // ticket #142, use a temporary working directory where it had previously used the mod installation directory.
+  QDir finalTargetModDirectory = QString::fromStdWString(QDir::toNativeSeparators(modInterface->absolutePath()).toStdWString().c_str());

```

Comment:
What's going on on this line? You seem to be converting stuff back and forth through a lot of formats for no reason.

Reply:
Thank you,  these two items are cleaned up and uploaded - the capitalization of the include name, and the string-conversion which wasn't needed.